packages/dates/src/utilities/formatDate.ts

```diff
@@ -50,8 +53,10 @@ export function formatDate(
 }
 
 function dateTimeFormatCacheKey(
-  locales: string | string[],
+  locales?: string | string[],
   options?: Intl.DateTimeFormatOptions,
 ) {
-  return `${locales}-${JSON.stringify(options)}`;
+  return `${
+    Array.isArray(locales) ? locales.join('-') : locales
```

Comment:
The same comment in this PR, I would prefer we move the ternary outside of the interpolation.